app/main.py

```
@@ -20,7 +20,7 @@
 from slowapi.errors import RateLimitExceeded  # noqa: E402
 from starlette.exceptions import HTTPException as StarletteHTTPException  # noqa: E402
 
-from app.api import admin, queue, rate, retros, roll, session, tasks, thread, undo  # noqa: E402
+from app.api import admin, auth, queue, rate, retros, roll, session, tasks, thread, undo  # noqa: E402
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)